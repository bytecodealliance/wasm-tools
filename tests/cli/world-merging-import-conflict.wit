// FAIL: component embed --dummy --world into % | component embed --world %from % | component wit

package a:b;

world into {
  import a: interface {}
}

world %from {
  import a: func();
}
