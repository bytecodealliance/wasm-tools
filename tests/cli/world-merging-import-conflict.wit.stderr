error: updating metadata for section component-type

Caused by:
    0: failed to merge worlds from two documents
    1: failed to merge world import a
    2: different kinds of items
